var/spack/repos/builtin/packages/py-archspec/package.py

```diff
@@ -15,8 +15,9 @@ class PyArchspec(PythonPackage):
 
     maintainers("alalazo")
 
+    version("0.2.2", sha256="d922c9fd80a5234d8cef883fbe0e146b381c449062c0405f91714ebad1edc035")
     version("0.2.1", sha256="0974a8a95831d2d43cce906c5b79a35d5fd2bf9be478b0e3b7d83ccc51ac815e")
     version("0.2.0", sha256="6aaba5ebdb5c3633c400d8c221a6a18716da0c64b367a8509f4217b22e91a5f5")
 
```

Comment:
Missing upper bound on Python 3. Also doesn't support all Python 3.6 versions if you want to start documenting that 😛